test/exchange/dfs-buy.js

```diff
@@ -82,7 +84,7 @@ describe('Dfs-Buy', function () {
                 buyAddr,
                 sellAmount,
                 buyAmount,
-                kyberWrapper.address,
+                uniWrapper.address,
```

Comment:
Old uniWrapper?

Reply:
running dfs-buy/ dfs-sell tests now will test both old uni wrapper and new uni wrapper